nss-cohort-36/bangazon-api-imagination-station-api
# Ts testing and added fields

# Description

Added functionality to add new fields when registering new customer.

## Type of change
Please delete options that are not relevant.
# Testing Instructions
1. fetch and checkout
1. Go to http://127.0.0.1:8000/register in postman
1. Put this in the body `{
	"username": "noobdude",
	"password": "noobpassword",
	"email": "noob@email.com",
	"first_name": "noob",
	"last_name": "dude",
	"address": "123 noob st.",
	"city": "New Noob City",
	"zipcode": "00000",
	"phone": "123-123-1234"
}`
1. Clear your token as you should be having one returned.
1. Checked db for new user is in  auth_user and customer
1. Do `python manage.py test`.
# Checklist:
